app/src/main/java/com/github/palFinderTeam/palfinder/utils/generics/FirestoreRepository.kt

```diff
@@ -59,11 +59,13 @@ class FirestoreRepository<T : FirebaseObject>(
 
     override suspend fun edit(uuid: String, field: String, value: Any): String? {
         if (!exists(uuid)) return null
+/*
```

Comment:
Might need to remove instead of commenting out

Reply:
or add a comment to explain why it must stay

Reply:
Fixed